.travis.yml

```diff
@@ -57,11 +64,15 @@ _helpers:
   os: osx
   language: generic
   python: *pypy3
-  env:
-  - &env_pypy3 PYTHON_VERSION=pypy3.6-7.1.1
-  - &env_pyenv PYENV_ROOT="$HOME/.pyenv"
-  - &env_path PATH="$PYENV_ROOT/bin:$PATH"
-  before_install:
+  env: &env_pyenv_base
+    PYTHON_VERSION: *pypy3
+    PATH: ${PYENV_ROOT}/bin:${PATH}
+  env: &env_osx_base
+    <<: *env_pyenv_base
+    PYTHON_VERSION: *pypy3
```

Comment:
I'd like to see a comment as to why this prefers to use a pypy3 image over other image options (such as `shell` or `generic` or `python`.